### ISSUE TITLE
VMATBase _median_profiles function returning mean profiles

### ISSUE THREAD

**Issue by keithoffer** [jrkerns/pylinac#483]
I'm unsure if I'm reading things incorrectly, but in the VMATBase class (used for DRMLC and DRGS analysis), as far as I can tell, the _median_profiles() method is actually returning mean profiles. In particular, [this line](https://github.com/jrkerns/pylinac/blob/dbb0693d23cc4eed5679ac09b1ed4c9409c41a9b/pylinac/vmat.py#L494) calls np.mean on the images, not np.median which is what I'd expect. However going back through the history, it looks like it's always been like this?

**Comment by jrkerns** [comment]
Yup, looks like a naming thing. Mean works better IMO in the event the field is narrower in the Y than the background. I.e. if the field were only 10cm tall on a 40cm EPID, the median would be the background value.
